### PR TITLE
ERC20 contract fixture.

### PR DESCRIPTION
**Pull request by g-r-a-n-t**
addresses #67 

I'll list the needed language features here and then create issues for each one.

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [ ] Create issues for the features mentioned below
- [x] Create a test that parses the contract
- [x] Clean up commit history

### CODE REVIEW

**Comment by g-r-a-n-t** [comment]
**create issues for the following features:**
- [ ] constructors
 - we should be able initialize storage values with a constructor
- [ ] string type
 - create a new *string* type
 - support the use of strings in storage
 - support the use of strings in memory
 - support string abi encoding 
- [ ] u8 type
 - create a new *u8* base type
 - support the use of u8 in storage
 - support the use of u8 in memory
 - support u8 abi encoding 
- [ ] u256 type
 - support the use of u256 in storage
- [ ] tuple type
 - create a new *Tuple* type
 - support the use of tuples in storage
 - support the use of tuples in memory
 - support tuple abi encoding
- [ ] assert statement support
- [ ] support for events with any set of parameters

File: compiler/tests/fixtures/erc20_token.fe [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        emit Transfer(sender, recipient, amount)
+
+    def _mint(account: address, amount: u256):
+        assert account != address(0)
+
+        self._beforeTokenTransfer(address(0), account, amount)
+
+        self._totalSupply = self._totalSupply + amount
+        self._balances[account] = self._balances[account] + amount
+        emit Transfer(address(0), account, amount)
```
Comment (cburgdorf) [inline compiler/tests/fixtures/erc20_token.fe:72]: Looks like `Transfer`  and `Approval` should be defined anywhere in this file.

Comment (g-r-a-n-t) [reply compiler/tests/fixtures/erc20_token.fe:72]: good catch :)

**Comment by g-r-a-n-t** [comment]
@cburgdorf let me know if the contract and feature list generally look good. I'm going to spend some time working on #77 now

**Review by cburgdorf — APPROVED** [review]
LGTM

**Pull request merged**